content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -290,6 +290,13 @@ spec:
   nfs:
     path: /tmp
     server: 172.17.0.2
+  nodeAffinity:
```

Comment:
@msau42 should i remove this section as it is not needed to be explicitly set for all volume types ?

Reply:
yes i think that would be best

Reply:
removed this section